src/js/redux/actions/agency/agencyActions.js

```diff
@@ -80,6 +80,15 @@ export const resetFederalAccountsList = () => ({
     type: 'RESET_FEDERAL_ACC_LIST'
 });
 
+export const setTasList = (agencyTasList) => ({
+    type: 'SET_TAS_LIST',
+    agencyTasList
+});
+
+export const resetTasList = () => ({
+    type: 'RESET_TAS_LIST'
```

Comment:
this case is not in the reducer and this function isn't used anywhere. is it still needed?